python-packages/smithy-event-stream/pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm", "wheel"]
+build-backend = "setuptools.build_meta"
```

Comment:
Are we using setuptools in the other projects? Not part of this PR but as a note, we should definitely move to [`hatchling`](https://hatch.pypa.io/latest/) over setuptools.

Reply:
We are - but we can switch. I have no strong opinions on the topic. Lets do it as one big changeover though